From: Artem Borisov <dedsa2002@gmail.com>
Date: Wed, 3 Feb 2021 12:49:05 +0000
Subject: Do not permit user-installed certificates

Current Android policy allows user-installed certificates usage only if that
specific certificate is permitted by the specific application in its network
security configuration.
Default Chromium configuration though allows all user-installed certificates,
potentially making it easier to let in the malicious traffic.
This patch disallows user certificates by default.
---
 chrome/android/java/res_base/xml/network_security_config.xml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/chrome/android/java/res_base/xml/network_security_config.xml b/chrome/android/java/res_base/xml/network_security_config.xml
--- a/chrome/android/java/res_base/xml/network_security_config.xml
+++ b/chrome/android/java/res_base/xml/network_security_config.xml
@@ -11,7 +11,6 @@
     See https://crbug.com/827265. See also MediaPlayerRenderer. -->
     <base-config cleartextTrafficPermitted="true">
         <trust-anchors>
-            <certificates src="user"/>
             <certificates src="system"/>
         </trust-anchors>
     </base-config>
-- 
2.17.1

